manifests/2.16.0/opensearch-2.16.0.yml

```diff
@@ -183,3 +183,9 @@ components:
       - anomaly-detection
       - sql
       - ml-commons
+  - name: query-insights
```

Comment:
Does query insights have any dependencies on other plugins?
I assume since it used to be part of core you dont need any additional dependencies.

Reply:
Yup no additional dependencies as of now